.github/workflows/maven.yaml

```diff
@@ -10,8 +10,12 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
 
+  # build the project, skip tests, update snapshots
```

Comment:
1. build the project, skip tests, update snapshots (_after_ the cache is restored)